public/modules/custom/grants_metadata/grants_metadata.module

```diff
@@ -164,6 +164,10 @@ function grants_metadata_field_industry_options(): array {
  */
 function grants_metadata_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   if ($form_id == 'webform_settings_form' || $form_id == 'webform_add_form' || $form_id == 'webform_duplicate_form') {
+
+    $currentUser = Drupal::currentUser();
+    $currentUserRoles = $currentUser->getRoles();
```

Comment:
Onko näitä rooleja tarkoitus käyttää johonkin?

Reply:
Katos jääny roikkumaan jostain vanhasta palikasta. Poistan.